MovieEnthusiast.Application/Common/Requests/GetMoviesRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿using MovieEnthusiast.Api.Requests;
+
+namespace MovieEnthusiast.Application.Common.Requests;
+
+public class GetMoviesRequest : PaginatedRequest
```

Comment:
My linter gives this a https://rules.sonarsource.com/csharp/RSPEC-2094/
And it is actually correct. There is no point of having this class, you can just use PaginatedRequest directly